docs/en/ufs/GlusterFS.md

```diff
@@ -66,3 +66,9 @@ To stop Alluxio, you can run:
 ```console
 $ ./bin/alluxio-stop.sh local
 ```
+
+## Contributed by Alluxio Community
+
+GlusterFS UFS integration is contributed and maintained by Alluxio community.
+Source code locates at [here](https://github.com/Alluxio/alluxio-extensions/tree/master/underfs/glusterfs).
+Welcome to submit pull requests to improve the integration!
```

Comment:
`Feel free submit pull requests to improve the integration and update the documentation [here](https://github.com/Alluxio/alluxio/edit/master/docs/en/ufs/GlusterFS.md) if any information is missing or out of date.`